src/firefly/java/edu/caltech/ipac/firefly/server/query/PackagerScript.java

```diff
@@ -0,0 +1,159 @@
+package edu.caltech.ipac.firefly.server.query;
+
+import edu.caltech.ipac.firefly.core.background.ScriptAttributes;
+import edu.caltech.ipac.firefly.data.DownloadRequest;
+import edu.caltech.ipac.firefly.data.FileInfo;
+import edu.caltech.ipac.firefly.data.ServerRequest;
+import edu.caltech.ipac.firefly.server.ServerContext;
+import edu.caltech.ipac.firefly.server.db.EmbeddedDbUtil;
+import edu.caltech.ipac.firefly.server.network.HttpServiceInput;
+import edu.caltech.ipac.firefly.server.packagedata.FileGroup;
+import edu.caltech.ipac.firefly.server.util.DownloadScript;
+import edu.caltech.ipac.firefly.server.util.Logger;
+import edu.caltech.ipac.table.DataGroup;
+import edu.caltech.ipac.table.MappedData;
+import edu.caltech.ipac.table.io.VoTableReader;
+import edu.caltech.ipac.util.download.URLDownload;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static edu.caltech.ipac.firefly.core.background.ScriptAttributes.*;
+import static edu.caltech.ipac.firefly.core.background.ScriptAttributes.URLsOnly;
+import static edu.caltech.ipac.util.FileUtil.getUniqueFileNameForGroup;
+
+@SearchProcessorImpl(id = "PackagerScript")
+public class PackagerScript extends FileGroupsProcessor {
```

Comment:
This class implementation is not right.  If it's a generic, reusable script generator, then it should not have any ObsCore logic.  If it's an ObsCore data packager, then it is named wrong and there is already an ObsCorePackager.